Lib/test/test_ast.py

```diff
@@ -790,14 +790,14 @@ class N2(ast.Num):
             n = N(42)
             self.assertEqual(n.n, 42)
             self.assertEqual(n.z, 'spam')
-            self.assertEqual(type(n), N)
-            self.assertTrue(isinstance(n, N))
-            self.assertTrue(isinstance(n, ast.Num))
-            self.assertFalse(isinstance(n, N2))
-            self.assertFalse(isinstance(ast.Num(42), N))
+            self.assertIs(type(n), N)
+            self.assertIsInstance(n, N)
+            self.assertIsinstance(n, ast.Num)
+            self.assertNotIsInstance(n, N2)
+            self.assertNotIsInstance(ast.Num(42), N)
```

Comment:
Nice :)

By the way, https://github.com/isidentical/teyit / https://pypi.org/project/teyit/ is a nice tool that can upgrade some of these automatically.